cheroot/errors.py

```diff
@@ -56,3 +57,24 @@ def plat_specific_errors(*errnames):
 if sys.platform == 'darwin':
     socket_errors_to_ignore.extend(plat_specific_errors('EPROTOTYPE'))
     socket_errors_nonblocking.extend(plat_specific_errors('EPROTOTYPE'))
+
+
+acceptable_sock_shutdown_error_codes = {
+    errno.ENOTCONN,
+    errno.EPIPE, errno.ESHUTDOWN,  # corresponds to BrokenPipeError in Python 3
```

Comment:
do both of these map to `BrokenPipeError`? maybe worth putting on their own line so the comment is unambiguous